main.js

```diff
@@ -0,0 +1,40 @@
+const renderer = Renderer()
+const frogGame = FrogGame()
+const timers = []
+
+$("#main-container").on("click", "#start", function() {
```

Comment:
When you start having large functions like this you should start to consider breaking things up. Keep your event listener separate and have your callback function outside

`const startGame = () => {
//the rest of your code will then go in here. It makes it a lot easier to read. 
//you can even divide your code in here again into another function
getTimer()
}`
`$("#main-container").on("click", "#start", startGame)`